config/tests/samples/create/samples.go

```diff
@@ -90,17 +90,42 @@ func getNamespaces(samples []Sample) []string {
 	return results
 }
 
-func RunCreateDeleteTest(t *Harness, unstructs []*unstructured.Unstructured, cleanupResources bool) {
+type CreateDeleteTestOptions struct {
+	// Create is the set of objects to create
+	Create []*unstructured.Unstructured
+
+	// Updates is the set of objects to update (after all objects have been created)
+	Updates []*unstructured.Unstructured
```

Comment:
* Where do we plan to store these updates (assuming they are going to be another set of YAML files with slight changes)? If they are going to be under `samples/resources/` we need to make sure they don't get rendered in the resource reference page.

* Also we are already testing updates using artifacts under `pkg/test/resourcefixture/testdata/basic`.
For example: https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/pkg/test/resourcefixture/testdata/basic/certificatemanager/v1beta1/certificatemanagercertificate/certificatemanagercertificatemanageddns

Would it be easier to directly enable mock for these test cases?

Reply:
Yes, this is leveraging the existing fixture/sample data.  Where those have updates, we will now do those updates.

I don't mind us trying to enable mocks with the "old" tests, they're just a little more complicated to deal with, so I found it easier to just build up a new set of tests for the mock tests.

Reply:
> Where those have updates, we will now do those updates.

Are you referring to data under `config/samples/resources` or other test data? I don't think we have any updates under `config/samples/resources`.

Reply:
You're right, looks like we only set Update on the fixtures test data, not the samples test data.